### ISSUE TITLE
Graphics formats

### ISSUE THREAD

**Issue by DL5SMB** [danielkucera/epaper-station#8]
Hi,

is there anywhere information on what graphic formats (width/height, color depth)
are needed for specific displays?

You are working with PNGs, Dmitry originally used 24bit-BMPs as input for his converter.

Aaron states in his Repository for the custom acces point:

- Chroma74: 640x384
- EPOP900: 360x480

According to the [data sheet](https://fccid.io/E2X-ST-GR2900N/User-Manual/User-manual-2771116) the SolumST-GR-2900 should be accepting files with 296x128 pixels. 

Nevertheless this is not working for me. The files gets transmitted, the Solum refreshed, but I get only a blank, white screen.
Since Aarons graphical Custom FW works, I guess my problem seems to be the file format?

**Comment by danielkucera** [comment]
Hi,

the image has to be <MAC>.png so png format and the app converts it to 1bit BMP here: https://github.com/danielkucera/epaper-station/blob/master/station.py#L197

The size should match the display size. Check if the generated bmp (it is saved in the working directory) has some content. If not, try to provide black/white source png.

**Comment by DL5SMB** [comment]
Hi, thanks for your answer.

It's seems I may have broken something in FW by now. 
The pixel size was correct and the BMP file was fine too. But I just got a blank screen.
I tried to switch back between different FW versions and now after checking for a suitable PNG or FW Update file nothing happens any more. The files do not get transmitted.

I switched between FW versions again (stock, precompiled, self compiled), used different pictures, a fresh installation on a another computer nothing.

What I noticed: 
imgUpdateVer shows always the same Value. Is this the counter in the tag which checks for changes? If yes, can it be overwritten?


_2022-08-23 23:46:32,029 Got checkin request
2022-08-23 23:46:32,029 CheckinInfo(swVer=1163936137216, hwType=8, batteryMv=2600, lastPacketLQI=77, lastPacketRSSI=192, temperature=151, rfu=b'\x00\x00\x00\x00\x00\x00')
2022-08-23 23:46:32,029 Reading image file: 3a980003fe9874cb.png
2022-08-23 23:46:32,030 Reading firmware file: UPDT0008.BIN
2022-08-23 23:46:32,030 No Firmware file available
2022-08-23 23:46:32,030 PendingInfo(imgUpdateVer=7133718651982842761, imgUpdateSize=5182, osUpdateVer=0, osUpdateSize=0, nextCheckinDelay=0, rfu=bytearray(b'\x00\x00\x00\x00'))
2022-08-23 23:59:43,772 Got checkin request
2022-08-23 23:59:43,773 CheckinInfo(swVer=1163936137216, hwType=8, batteryMv=2600, lastPacketLQI=70, lastPacketRSSI=191, temperature=151, rfu=b'\x00\x00\x00\x00\x00\x00')
2022-08-23 23:59:43,773 Reading image file: 3a980003fe9874cb.png
2022-08-23 23:59:43,773 Reading firmware file: UPDT0008.BIN
2022-08-23 23:59:43,773 No Firmware file available
2022-08-23 23:59:43,774 PendingInfo(imgUpdateVer=7133718651982842761, imgUpdateSize=5182, osUpdateVer=0, osUpdateSize=0, nextCheckinDelay=0, rfu=bytearray(b'\x00\x00\x00\x00'))
2022-08-24 00:12:55,692 Got checkin request
2022-08-24 00:12:55,693 CheckinInfo(swVer=1163936137216, hwType=8, batteryMv=2600, lastPacketLQI=85, lastPacketRSSI=191, temperature=151, rfu=b'\x00\x00\x00\x00\x00\x00')
2022-08-24 00:12:55,693 Reading image file: 3a980003fe9874cb.png
2022-08-24 00:12:55,693 Reading firmware file: UPDT0008.BIN
2022-08-24 00:12:55,693 No Firmware file available
2022-08-24 00:12:55,709 PendingInfo(imgUpdateVer=7133718651982842761, imgUpdateSize=5182, osUpdateVer=0, osUpdateSize=0, nextCheckinDelay=0, rfu=bytearray(b'\x00\x00\x00\x00'))
2022-08-24 00:17:27,477 Got checkin request
2022-08-24 00:17:27,477 CheckinInfo(swVer=1163936137216, hwType=8, batteryMv=2600, lastPacketLQI=71, lastPacketRSSI=191, temperature=151, rfu=b'\x00\x00\x00\x00\x00\x00')
**2022-08-24 00:17:27,477 Reading image file: 3a980003fe9874cb.png
2022-08-24 00:17:27,478 Reading firmware file: UPDT0008.BIN**
**2022-08-24 00:17:27,478 PendingInfo(imgUpdateVer=7133718651982842761, imgUpdateSize=5182, osUpdateVer=2252907915247616, osUpdateSize=33127, nextCheckinDelay=0, rfu=bytearray(b'\x00\x00\x00\x00'))**_

**Comment by DL5SMB** [comment]
It remains a mystery to me.

To make sure my test display  is not damaged I uploaded the firmware to a second tag. 

Then again I tried with different PNG-pictures 296x128 (one example as attachment)

Both tags just switch to a white screen. 

As long as the "epaper-station" is running, the white screen remains, if i turn it off,
the tags switch back to the "Not-Connected"-FW screen after a certain time.

As the graphics seem to be okay, I can only imagine a problem with the transmission of the picture to the tag.
But the log does not show anything suspicious..
[graphics.zip](https://github.com/danielkucera/epaper-station/files/9438830/graphics.zip)


[Log.txt](https://github.com/danielkucera/epaper-station/files/9438823/Log.txt)
![20220827_210409](https://user-images.githubusercontent.com/29584431/187051598-909dff52-9a65-4bf3-a388-bc59571c8efc.jpg)

![20220827_221203](https://user-images.githubusercontent.com/29584431/187051602-3b7ce042-86fa-4337-bde0-00611c30fe1b.jpg)
